modul/achievements.py

```
@@ -4,34 +4,58 @@
 
 
 class Achievement:
+<<<<<<< HEAD
```

Pylint (4.0.4)

[error] Parsing failed: 'expected an indented block after class definition on line 6 (modul.achievements, line 7)'

(E0001)